### ISSUE TITLE
Simple rendering example

### ISSUE THREAD

**Issue by NaakkaDev** [aalmik/tmx-mapper-pcl#20]
I know that this is just a library for parsing Tiled map editor's TMX files, but that doesn't mean there shouldn't be an short example which shows how this lib can be used to render the map.

**Comment by aalmik** [comment]
You are right. It's good to have example to show how to use this parser in real project. I have currently implemented map renderer as an extension method class in my game development project and will share renderer very soon.

**Comment by aalmik** [comment]
I have created map rendering sample Project. MonoGame development build is needed.
http://www.monogame.net/downloads/
@Temeez Could you test?

**Comment by NaakkaDev** [comment]
I would, but there is something which requires Windows 8.1, I have Windows 7. Or at least that's what Visual Studio 2013 says when adding as existing project.
